crimsobot/utils/cringo.py

```diff
@@ -450,6 +497,17 @@ def line_score(direction: str, index: int = 0) -> bool:
 
         ess = '' if s.plays == 1 else 's'
         ess2 = '' if s.wins == 1 else 's'
+        ess3 = '' if s.full_cards == 1 else 's'
+
+        def dec() -> float:
```

Comment:
I went a little overboard here, but I wanted to see if it could be done. (It can!)

Reply:
I mean, that’s a fine experiment, but I don’t see why it’s necessary. I’d rather not do nested functions unless there’s an actual use-case.